config/activitylog.php

```diff
@@ -42,4 +42,10 @@
      * used by the Activity model shipped with this package.
      */
     'table_name' => 'activity_log',
+
+    /*
+     * This is the database connection that will be used by the migration and
+     * the Activity model shipped with this package.
+     */
+    'database_connection' => env('ACTIVITY_LOGGER_DB_CONNECTION', 'mysql'),
```

Comment:
I wonder if this should default to `DB_CONNECTION` instead of `mysql`, since a lot of unit tests set only `DB_CONNECTION` in `phpunit.xml` and therefore test suites will break without explicitly setting this config. Case in point, sqlite :memory: connections.

Reply:
Good point - now the philosophic question: should we use another `env()` in the fallback or just use `DB_CONNECTION`?

Reply:
https://github.com/spatie/laravel-activitylog/releases/tag/3.7.1

Reply:
Thanks @drbyte to pointed out. I was in doubt when I did the pull request but I think this is best solution! 
I will think twice about this config. It'll be helpfull for future projects! 👍